pom.xml

```diff
@@ -5,27 +5,86 @@
 
   <groupId>com.google.cloud.tools.ide.login</groupId>
   <artifactId>google-ide-login</artifactId>
-  <version>0.0.0-SNAPSHOT</version>
+  <version>0.1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <name>Login Support for Google Cloud Tools</name>
+  <name>IDE Login Support for Google Cloud Tools</name>
```

Comment:
Should we change the Maven group ID and the artifact ID first then? They are currently `com.google.cloud.tools.ide.login` and `google-ide-login`. The repo name is also `ide-login`.